app/views/prison_mailer/booking_cancellation_receipt_email.text.erb

```diff
@@ -2,6 +2,10 @@ Dear staff,
 
 <%= @visit.visitors.first.full_name %> has cancelled the visit to <%= @visit.prisoner.full_name %> (<%= @visit.prisoner.number %>) on <%= visit.slots.first.date %> <%= visit.slots.first.times %>.
 
+<% if @visit.vo_number %>
+Reference number: <%= @visit.vo_number %>
+<% end %>
+
```

Comment:
Please could you move this space inside the `if` block so it doesn't create a double space when there is no reference number?